plenum/test/consensus/view_change/test_sim_view_change.py

```diff
@@ -28,6 +29,12 @@ def filter(request):
     return request.param[0], request.param[1]
 
 
+@pytest.fixture(params=Random().sample([seed for seed in range(1000000)
```

Comment:
I've done a second pass on this PR.  I'm not seeing this change in either the latest version on `master` or `stable`.  Do we know why this change is being introduced?

Reply:
This is a change in tests because of issue https://github.com/hyperledger/indy-plenum/issues/1506 (view change bug)

Reply:
But that change is not showing up in either the latest in `master` or `stable`, so I would not expect this to show up as a change being introduced:
- https://github.com/hyperledger/indy-plenum/blob/master/plenum/test/consensus/view_change/test_sim_view_change.py
- https://github.com/hyperledger/indy-plenum/blob/stable/plenum/test/consensus/view_change/test_sim_view_change.py

Reply:
The only difference between `test_sim_view_change.py` on `master` and `stable` that I see is a comment on line 64/65:
![image](https://user-images.githubusercontent.com/10504350/110796286-a5658d80-822c-11eb-8608-29e31bc22531.png)

Reply:
I mean this fixture changing 
https://github.com/hyperledger/indy-plenum/pull/1515/files#diff-73a33258f86992e3bc2eb3163037d5a344f6e0f75cde333f93ed37a7194d61b9R72

Reply:
Right, but even that change does not exist in the latest on `master` or `stable`.